### ISSUE TITLE
If not, why not give this project to the community for maintenance?

### ISSUE THREAD

**Issue by windsOne** [Azure/DotNetty#495]
如果不想维护的话，可否直接交给社区维护~

If not, why not give this project to the community for maintenance?

**Comment by windsOne** [comment]
@caozhiyuan
@yyjdelete

**Comment by caozhiyuan** [comment]
nayato is responsible to this project , and this project is stable. 

you can @ nayato 

@windsOne

**Comment by fanliang11** [comment]
I don't think it's a stable problem. We need more protocol extensions. The open source project surging is also based on dotnetty. I also want dotnetty to be as active as netty.

**Comment by windsOne** [comment]
@nayato @caozhiyuan 
I think so

> I don't think it's a stable problem. We need more protocol extensions. The open source project surging is also based on dotnetty. I also want dotnetty to be as active as netty.

**Comment by InCerryGit** [comment]
> I don't think it's a stable problem. We need more protocol extensions. The open source project surging is also based on dotnetty. I also want dotnetty to be as active as netty.

I agree

**Comment by ScarletKuro** [comment]
I also agree. There are a lot of pull requests that are just hanging. There is still lot of stuff to implement like compression etc.

**Comment by onatm** [comment]
I strongly believe that we should find a way to make this project more active. Last year I started to build an experimental web framework on top of dotnetty but due to the inactivity of the project I decided not to waste time on it and moved to rust programming language...

**Comment by caozhiyuan** [comment]
@nayato  consider give this project to the community for maintenance? may be can consider 
 dotnetfoundation

**Comment by brucehu123** [comment]
agreeeeee

**Comment by stratosblue** [comment]
> 
> 
> I don't think it's a stable problem. We need more protocol extensions. The open source project surging is also based on dotnetty. I also want dotnetty to be as active as netty.

I think so

**Comment by gsw945** [comment]
@nayato

**Comment by linfx** [comment]
why stop it

**Comment by gsw945** [comment]
Maintenancer has no response until now, but I found [SpanNetty](https://github.com/cuteant/SpanNetty), it's an active  fork from [DotNetty](https://github.com/Azure/DotNetty)
